Assignment2/fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+	"@Context":{
+		"fullname": "http://www.upm/fullname/" ,
+		"hometown": "http://www.upm/hometown/" ,
+		"attributes": "http://www.upm/attributes/" ,
+ 		"age" : "attributes: age" ,
+		"almamater": "attributes: name" ,
+ 		"parents": "attributes: name" ,

```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your almamater, if you query "attributes:name" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "almamater": "attributes:almamater". This comment also applies to the parents attribute.